releases/note-source/v15_0.rst

```diff
@@ -0,0 +1,239 @@
+.. _release-v15-0:
+
+Spring 2018 Release (v15_0)
+===========================
+
+.. ATTENTION::
+   This will be the last major release that supports Python2. From release v16.0 (Fall 2018) on, only Python3 be explicitly supported. Minor releases of v15 will continue to support Python2.
+
++-------------------------------------------+------------+
+| Source                                    | Identifier |
++===========================================+============+
+| Git tag                                   | 15.0       |
++-------------------------------------------+------------+
+| :doc:`EUPS distrib </install/newinstall>` | v15\_0     |
++-------------------------------------------+------------+
+
+- :ref:`release-v15-0-major-changes`
+- :ref:`release-v15-0-bug-fixes`
+- :ref:`release-v15-0-internal-improvements`
+
+.. - :ref:`release-v15-0-sui`
+.. - :ref:`Qserv and Data Access <release-v15-0-qserv-dax>`
+
+*See also:*
+
+- :doc:`Installation instructions <../install/index>`
+- :doc:`Known issues </known-issues>`
+- `Characterization Metric Report (DMTR-62) <https://ls.st/DMTR-62>`_
+
+These release notes highlight significant changes to the Science Pipelines codebase which are likely to be of wide interest.
+For a complete list of changes, see :ref:`release-v15-0-tickets`.
+
+.. _release-v15-0-major-changes:
+
+Major Functionality and Interface Changes
+-----------------------------------------
+- :ref:`release-v15-0-diffim`
+- :ref:`release-v15-0-mem`
+- :ref:`release-v15-0-artifacts`
+- :ref:`release-v15-0-skysub`
+- :ref:`release-v15-0-wcs`
+- :ref:`release-v15-0-firefly`
+- :ref:`release-v15-0-compression`
+- :ref:`release-v15-0-cmdline`
+- :ref:`release-v15-0-prereqs`
+
+
+
+.. _release-v15-0-diffim:
+
+Image differencing algorithm improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Algorithms appropriate for differencing in a variety of contexts have been implemented as tasks in the LSST stack. These various algorithms and configurations have been compared.  The results are written up in `DMTN-061 <https://dmtn-061.lsst.io>`_.
```

Comment:
The previous release also came with ip_diffim. If possible, it'd be helpful to provide a little more context about what's actually new here.

Reply:
It's mostly the writeup that explains the various algorithms that is new.  I don't know how to call that out more clearly.